bc_obps/compliance/templates/partials/elicensing_stale_data_alert.html

```diff
@@ -0,0 +1,49 @@
+<style id="bciers-elicensing-alert-styles">
+  .bciers-alert {
+    display: flex;
+    align-items: flex-start;
+    gap: 8px;
+    background: #fff7e0;
```

Comment:
In the app, the alert background is light blue, and in the invoices, it's yellow. Are they meant to be different?